src/components/Footer.astro

```diff
@@ -0,0 +1,52 @@
+---
+const {} = Astro.props;
+---
+
+<div class="footer-wrapper">
+  <!-- footer wrapper -->
+  <hr />
+  <div class="container">
+    <footer>
+      <ul class="list-inline pull-right">
+        <li style="position: relative; top: 5px">
```

Comment:
because it is used in several places, it's better to make this a component-wide style for `<li>`

Reply:
I can only find it in two places? I'd rather leave this to a wider refactoring effort at a future date TBD to be honest..